### PR TITLE
CloudEvents sandbox proposal

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by caniszczyk** [comment]
RFC @cncf/toc, this will be presented at the next TOC meeting on May 15th along with an update from the serverless wg

File: proposals/cloudevents.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+- a [whitepaper](https://github.com/cncf/wg-serverless#whitepaper) that:
+  - defines Serverless and its terminology
+  - describes common use cases for the technology
+  - compares it with other Cloud Native technologies and **aaS environments
+  - describes the common architecture of Serverless platforms
+- a [landscape document](https://docs.google.com/spreadsheets/d/10rSQ8rMhYDgf_ib3n6kfzwEuoE88qr0amUPRxKbwVCk/edit#gid=0)
+  that lists well-known open-source and proprietary Serverless platforms
+  and tools
+- a set of recommended next steps for the WG, as part of the whitepaper:
+  - encourage more serverless technology vendors and open source developers
```
Comment (barkerd427) [inline proposals/cloudevents.md]: Are we capitalizing serverless all the time? This is inconsistent with previous usage above, but I don't know the idiom here.

File: proposals/cloudevents.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+Events that are transferred between an event producer and an event consumer
+should be formalized. The purpose of this would be to better enable
+interoperability between these components such that basic processing of
+the events (such as routing) can be achieved without having to require
+knowledge of the event's structure in advance, or without understanding
+the application specific data of the event.
+
+The work on this specification is currently being done within the
+CNCF's Serverless Working Group, but with the release of our first
+milestone release (v0.1), it would make sense for this work to be more
```
Comment (barkerd427) [inline proposals/cloudevents.md]: Is release redundant? I'm not sure in this context.

File: proposals/cloudevents.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  to enable broad adoption within the Cloud community, and in particular
+  for Serverless/FaaS implementations
+- define protocol mappings for popular transports, such as HTTP
+- define serialization mappings for popular formats, such as JSON
+
+**Statement on alignment with CNCF mission**:
+
+Being born out of the CNCF's Serverless Working Group, the CloudEvents
+project (and its members) share the CNCF's goals of promoting Cloud Native
+technologies, and offering choice to our consumers though open interoperability
```
Comment (barkerd427) [inline proposals/cloudevents.md]: ...th**r**ough **an** open... instead of ...though open...

File: proposals/cloudevents.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+specification, as shown by the significant participation from key industry
+companies.
+
+We believe that the CNCF provides the proper home for this due to its
+committment to the promotion, and development, of open vendor-neutral projects.
```
Comment (barkerd427) [inline proposals/cloudevents.md]: "commitment to the promotion and development of open, vendor-neutral projects." spelling and punctuation.

File: proposals/cloudevents.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+**Social Media Accounts**:
+
+Twitter: @CloudEventsDemo 
+
+**Contributor statistics**:
+
+Attendence is tracked [here](https://docs.google.com/spreadsheets/d/1bw5s9sC2ggYyAiGJHEk7xm-q2KG6jyrfBy69ifkdmt0/edit?pli=1#gid=0).
+As can been seen in that document, CloudEvents weekly calls have regular
+attendence from most major cloud vendors, averaging nearly 30 people
```
Comment (barkerd427) [inline proposals/cloudevents.md]: attendance

File: proposals/cloudevents.md (hunk 1 above)
Comment (duglin) [reply proposals/cloudevents.md]: I went with making it "S" :-)

File: proposals/cloudevents.md (hunk 2 above)
Comment (duglin) [reply proposals/cloudevents.md]: removed

File: proposals/cloudevents.md (hunk 3 above)
Comment (duglin) [reply proposals/cloudevents.md]: fixed

File: proposals/cloudevents.md (hunk 4 above)
Comment (duglin) [reply proposals/cloudevents.md]: fixed

File: proposals/cloudevents.md (hunk 5 above)
Comment (duglin) [reply proposals/cloudevents.md]: got all 3 of 'em

**Comment by duglin** [comment]
@barkerd427 updated per your edits, thanks!

File: proposals/cloudevents.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+from key industry companies.
+
+We believe that the CNCF provides the proper home for this due to its
+commitment to the promotion and development of open, vendor-neutral projects.
+Additionally, the wide breadth of the CNCF members will provide the feedback
+necessary to ensure the CloudEvents specification isn't too limited in its
+scope and appeals to as many constituents of the cloud native community
+as possible.
+
+**Sponsor / Advisor from TOC**: Ken Owens <ken.owens.mastercard.com>
```
Comment (bgrant0607) [inline proposals/cloudevents.md]: I'm happy to co-sponsor

**Comment by caniszczyk** [comment]
@bgrant0607 @kenowens12 can you confirm here you're OK sponsoring cloudevents and we will move forward with to the sandbox

**Comment by caniszczyk** [comment]
Here you go I got a record of Brian and Ken sponsoring on the TOC call, we can use that, just need something formally recorded: https://www.youtube.com/watch?v=PjLM9ZhJJ1k&feature=youtu.be

**Comment by caniszczyk** [comment]
Welcome CloudEvents to the sandbox, we will work with you to get you up to speed on being part of the CNCF project family!

**Comment by duglin** [comment]
woo hoo! thanks!!

**Comment by bgrant0607** [comment]
Late, but confirming.

**Pull request merged**